Marlin/src/lcd/ultralcd.cpp

```diff
@@ -791,6 +773,21 @@ void MarlinUI::update() {
           quick_feedback();                             //  - Always make a click sound
         }
       }
+      else if (buttons & (EN_A | EN_B)) {               // Ignore the encoder if clicked, to prevent "slippage"
+        const millis_t ms = millis();
+        if (ELAPSED(ms, next_button_update_ms)) {
+          next_button_update_ms = ms + 50;
+          encoderDiff = ENCODER_STEPS_PER_MENU_ITEM * ENCODER_PULSES_PER_STEP;
+          if (buttons & EN_A) encoderDiff *= -1;
+          if (!wait_for_unclick) {
+            next_button_update_ms += 250;
+            #if HAS_BUZZER
+              buzz(LCD_FEEDBACK_FREQUENCY_DURATION_MS, LCD_FEEDBACK_FREQUENCY_HZ);
```

Comment:
btw, with a buzzer the LCD_FEEDBACK_FREQUENCY_DURATION_MS may affect the next_button_update_ms ... Maybe a smaller ms value than validation one should be used on +/-

Reply:
Most buzzers are non-blocking. I guess we need to add a test for that before applying the `MIN`.